app-text/sigil/sigil-0.9.17.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+PYTHON_REQ_USE="tk"
+
+inherit cmake-utils python-single-r1 readme.gentoo-r1 xdg-utils
+
+DESCRIPTION="Multi-platform WYSIWYG ebook editor for ePub format"
+HOMEPAGE="https://sigil-ebook.com/"
+SRC_URI="https://github.com/Sigil-Ebook/Sigil/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+ Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+plugins system-mathjax"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	app-text/hunspell:=
+	dev-libs/boost:=[threads]
+	dev-libs/libpcre:3=[pcre16]
+	dev-libs/xerces-c[icu]
+	dev-python/css-parser[${PYTHON_USEDEP}]
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	>=dev-qt/qtconcurrent-5.12:5
+	>=dev-qt/qtcore-5.12:5
+	>=dev-qt/qtgui-5.12:5
+	>=dev-qt/qtprintsupport-5.12:5
+	>=dev-qt/qtwebengine-5.12:5[widgets]
+	>=dev-qt/qtwidgets-5.12:5
+	>=dev-qt/qtxmlpatterns-5.12:5
+	sys-libs/zlib[minizip]
+	plugins? (
+		dev-python/chardet[${PYTHON_USEDEP}]
+		dev-python/cssselect[${PYTHON_USEDEP}]
+		dev-python/cssutils[${PYTHON_USEDEP}]
+		dev-python/html5lib[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/regex[${PYTHON_USEDEP}]
+	)
+	system-mathjax? ( dev-libs/mathjax )
+"
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
+
+S="${WORKDIR}/Sigil-${PV}"
+
+DOCS=( ChangeLog.txt README.md )
+
+DOC_CONTENTS="From Sigil's release notes:
+When you fire up Sigil for the very first time:
+navigate to the new General Preferences and select the default
+epub version you plan to work with (epub 2 or epub3) so that new
+empty ebooks start with the correct code.
+if you plan to work with epub3 epubs, you should change your
```

Comment:
Thanks! I only noticed `epub 2` vs. `epub3` but rest of it looked good to me so it wasnt a blocker. Compiled and runs good.